scripts/ci/lib.sh

```diff
@@ -810,7 +808,11 @@ check_rhacs_eng_image_exists() {
     local url="https://quay.io/api/v1/repository/rhacs-eng/$name/tag?specificTag=$tag"
     info "Checking for $name using $url"
```

Comment:
I'm wondering if we should maybe print a warning to `stderr` if token is empty and `$name` is something else than the index image 🤔 
This will require changes if we ever make more images public 👎🏻 , but it will decrease the surprise factor for anyone who tries to use this function without a token on a different image 👍🏻 . WDYT?

Reply:
added